policies/loading-frame-default-eager.md

```diff
@@ -0,0 +1,46 @@
+loading-frame-default-eager Policy
+===========
+
+The `loading-frame-default-eager` policy-controlled feature can be used for a main frame or a nested
+`<iframe>` to overwrite the default or assigned value of the newly
+introduced `loading` [attribute](https://github.com/whatwg/html/pull/3752) for `<iframe>` and `<img>`.
+
+What does that mean?
+------------
+With the `loading-frame-default-eager` attribute, developers could prioritize the loading of different inline frames on a web page. This however could become a cumbersome process and not quite scalable for larger
+web sites specially given that applying the attribute is origin-agnostic. The proposed policy aims to resolve
+this issue but changing a browser's default decision for `loading` behavior.
+
+Proposed Solution
+------------
+The `loading` attribute can take one of the three values:
+  * **`eager`** indicating that the resource should load eagerly (i.e., load even when below the fold).
+  * **`auto`** the browser makes the call for the loading behavior; it probably defaults to `eager`.
+  * **`lazy`**: the resource loads lazily.
+
+The proposed `loading-frame-default-eager` policy will affect the `auto` attribute value in that when disabled inside a document, all the `<iframe>`'s which do not explicitly set the `loading` attribute to `eager` will most probably load lazily. This is equivalent to overriding the `loading` attribute of `auto` with `lazy`.
```

Comment:
I'd switch these two things around -- when disabled, auto (or unset) will be treated as lazy, which means that frames which don't set the attribute to `eager` will (usually) be lazily loaded.